storops/unity/resource/lun.py

```diff
@@ -437,12 +451,12 @@ def _do_check_move_session(move_session_id):
                 return False
 
         clz = storops.unity.resource.move_session.UnityMoveSession
-        if is_compressed not in (True, False):
-            is_compressed = self.is_data_reduction_enabled
         try:
-            move_session = clz.create(self._cli, self, dest,
-                                      is_data_reduction_applied=is_compressed,
-                                      is_dest_thin=is_thin)
+            move_session = clz.create(
+                self._cli, self, dest,
+                is_data_reduction_applied=is_compressed,
+                is_dest_thin=is_thin,
+                is_advanced_dedup_enabled=is_advanced_dedup_enabled)
```

Comment:
`is_advanced_dedup_enabled`? It should be `is_advanced_dedup_applied`? Did pep8 pass?

Reply:
Please skip this commit, it is not fully tested.

Reply:
Fixed.